src/pfe/portal/modules/ExtensionList.js

```diff
@@ -42,7 +42,12 @@ module.exports = class ExtensionList {
           await extension.initialise();
           this.add(extension);
           if (extension.templates)
-            templates.addRepository(extension.templates, extension.description);
+            try {
```

Comment:
Actually we need the `try` right under the `for` loop, as `await fs.lstat` and `await extension.initialise` above could both throw error